### ISSUE TITLE
Compiling Errors

### ISSUE THREAD

**Issue by brehmupmc** [Subito-it/SBTUITestTunnel#88]
I had some issues getting 6.0.0 to compile for my project as well as the Example project. After some digging I found that there was some additional code that needed to be added.

SBTStubResponse.swift
* most of the get {..} need to have a return
`@objc public static var defaultReturnCode: Int {
        get { return defaults.returnCode }
        set { defaults.returnCode = newValue }
    }`

SBTUITestTunnelServer.swift
*  XCUITest CLLocation extensions needed to be wrapped in 
`#if ENABLE_UITUNNEL && ENABLE_UITUNNEL_SWIZZLING`

*  XCUITest UNUserNotificationCenter extensions needed to wrapped in 
`#if ENABLE_UITUNNEL && ENABLE_UITUNNEL_SWIZZLING`

I also updated the podfile to add in the ENABLE_UITUNNEL_SWIZZELING in the "post_install do..."
`
config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= ['$(inherited)', 'ENABLE_UITUNNEL=1', 'ENABLE_UITUNNEL_SWIZZLING=1']
`

**Comment by tcamin** [comment]
I tried cloning the project and run tests as follows:

```bash
❯ cd /tmp
❯ git clone https://github.com/Subito-it/SBTUITestTunnel.git
❯ xed SBTUITestTunnel/Example

// Hit ⌘+U and tests run as expected
```

Does the above work for you?

I just released version 6.1.0 which contains a few commits with some fixes that might eventually fix your problem.

**Comment by brehmupmc** [comment]
What Swift version are you compiling on?

**Comment by tcamin** [comment]
I'm using Xcode 11.1 (Swift 5.1).

**Comment by brehmupmc** [comment]
ah...we are on 5.0.
5.0 requires explicit returns in getters.
as of 5.1 you dont need them 

https://docs.swift.org/swift-book/LanguageGuide/Properties.html

Currently we are forced to use Swift 5.0 for 3rd party compatibility.
Is it possible to add in the explicit returns in the getters (as posted above) for backwards compatability with Swift 5.0?

**Comment by tcamin** [comment]
Ups... We're integrating the library in our main project and Swiftlint is the culprit 😇.

Makes total sense to guarantee backward compatibility, gonna fix this asap.

**Comment by brehmupmc** [comment]
Thanks! 👍

**Comment by tcamin** [comment]
Just released 6.2.0. Should be fixed
